docs/build_system.rst

```diff
@@ -47,7 +47,7 @@ To install all dependencies in one step, use:
 
 .. code-block:: bash
 
-    pip install . --no-build-isolation --no-install
+    pip install 'setuptools_scm[toml]>=4' 'cython' 'cmake>=3.24.2,<3.28' 'setuptools-rust<2'
```

Comment:
This could be simplified with `pip install '.[build-system]'`

Reply:
@emmettbutler The command appears to work but I see the following warning:
```
WARNING: ddtrace 3.6.0.dev16+gb2d78cefb does not provide the extra 'build-system'
```

Also the point of the command seemed to be installed deps without building the project, but that actually builds it.